src/domcoords.js

```diff
@@ -283,16 +283,18 @@ export function coordsAtPos(view, pos) {
   let {node, offset} = view.docView.domFromPos(pos)
 
   // These browsers support querying empty text ranges
-  if (node.nodeType == 3 && (browser.chrome || browser.gecko)) {
+  if (node.nodeType == 3 && browser.webkit) {
```

Comment:
You're turning off this branch for Gecko, which would make the `if (browser.gecko ...)` condition below always false, no?

Reply:
Right. I have access only to Firefox 70.0.1 at that version is showing browser.webkit true on Win7 VM. Nevertheless, we should indeed add gecko browser condition here.

Reply:
Ah, lovely, Firefox has implemented the webkit-prefixed style I used for browser detection. Going to have to come up with something else there.